akka-http-jwt/src/main/resources/reference.conf

```
@@ -0,0 +1,40 @@
+#######################################
+# akka-http-jwt Reference Config File #
+#######################################
+
+# This is the reference config file that contains all the default settings.
+# Make your edits/overrides in your application.conf.
+
+akka.http.jwt {
+  # Enables or disables the JWT signature validation.
+  # This is useful for development and testing purposes
+  # where you can still assert presence of claims without validating the signature.
+  dev = off
+
+  # The realm to use in the WWW-Authenticate header when a token is missing or invalid.
+  realm = "akka-http-jwt"
+
+  # Allows configuration for the JWT secrets used to verify tokens.
```

Should we have more of an explicit doc listing of the supported algorithms and what config keys each require or do you think an example section covering all like now is enough?

I'm thinking something like
```suggestion
  # Allows configuration for the JWT secrets used to verify tokens.
  # Supported algorithms and their required key configs:
  #  - "HS256" - requires either a secret in 'secret' or a filesystem path with a secret via 'secret-path', if both are present xxx happens
```

---

Yeah, I think it makes sense. Added now, not exactly in that format.. but have a look and let me know: https://github.com/akka/akka-http/pull/4377/commits/51f09bfe5c1eb04f807bb1887a140592e95eb0aa